src/web/handlers/GestureHandler.ts

```diff
@@ -44,47 +39,26 @@ export default abstract class GestureHandler {
   protected shouldResetProgress = false;
   protected pointerType: PointerType = PointerType.NONE;
 
-  public constructor() {
-    this.hasCustomActivationCriteria = false;
+  protected delegate: GestureHandlerDelegate<unknown>;
```

Comment:
Why can't we do

```suggestion
  protected delegate: GestureHandlerDelegate<HTMLElement>;
```

Reply:
Same as above - we can, but shouldn't if we want to use the js implementation on different platforms.